docs/release-notes/5.7.44-50.md

```diff
@@ -0,0 +1,68 @@
+# Percona Server for MySQL 5.7.44-50 (2024-06-19)
+
+This release is part of {{post}}. The fixes are available to [MySQL 5.7 Post-EOL Support from Percona customers]. Community members can [build this release from the source]. We recommend that you upgrade to MySQL 8.0 or you can stay on 5.7; we'll support you either way.
+
+## Release Information
+
+Percona Server for MySQL 5.7.44-50 contains the following fixes.
```

Comment:
The word "security" is not mentioned at all in the entire page. This looks strange to me as we are delivering security fixes with this version (part of the EOL program).  I would replace this sentence with:

"Percona Server for MySQL 5.7.44-50 includes all the features and bug fixes available in MySQL 5.7.44 Community Edition in addition to enterprise-grade features and security fixes developed by Percona."